### DISCUSSION TITLE
Question on mystRole nodes and where to edit the default text in https://myst-tools.org/sandbox

### DISCUSSION THREAD

**Discussion by Carreau** [executablebooks/meta#952] [General]
Hello I have two questions:

First one with respect to mystRole in the form `` {name}`text` ``. Sphinx/(docutils?) seem to distinguish domain/role with the `:`. myst goes simply with name. I'm guessing this is on purpose and I don't believe you have any reasons to track domain and role separately ? I haven't been able to find a case in rst-to-myst where there is a directive with explicit domain and role.

I guess if I'm generating mystRole from my own rst parsing that have an explicit domain, I can just use `domain+':'+role` and the mystRole name ? 

2) I was unable to find where the default text in https://myst-tools.org/sandbox is, I figured that mystRoles example with for example `` {ref}`...` `` was missing and could have added one.

**Comment by chrisjsewell** [comment]
Heya, well in myst-parser it literally just pipes this directly through to docutils/sphinx: https://github.com/executablebooks/MyST-Parser/blob/7692f87610976a61af522aef663576f11f600237/myst_parser/mdit_to_docutils/base.py#L1488

so there its handled exactly the same

**Reply by chrisjsewell** [reply]
the logic for sphinx is here: https://github.com/sphinx-doc/sphinx/blob/30f347226fa4ccf49b3f7ef860fd962af5bdf4f0/sphinx/util/docutils.py#L284

**Reply by chrisjsewell** [reply]
But for mystjs, no at the moment it essentially always assumes its the `std` domain, i.e. `{std:name}`
(well basically there is no "concept" of domains in mystjs)
